pages/index.tsx

```diff
@@ -31,7 +31,7 @@ import GradleLogo from '../public/images/companies/gradle.svg';
 import UberLogo from '../public/images/companies/uber.svg';
 import AtlassianLogo from '../public/images/companies/atlassian.svg';
 import GoogleLogo from '../public/images/companies/google.svg';
-import JetbrainsLogoSquare from '../public/images/companies/jetbrains-logo-square.svg';
+import JetbrainsLogo from '../assets/jetbrains-logo.svg';
```

Comment:
junie: use logo from the attachment and name it `jetbrains-logo-light.svg`

![JetBrains_company_logo](https://github.com/user-attachments/assets/4252deb9-5e1c-4625-9723-c8b97c1db0b0)